packages/server-side-graphql-client/README.md

```diff
@@ -0,0 +1,392 @@
+<!--[meta]
+section: api
+title: Server-side graphQL client
+[meta]-->
+
+# Server-side graphQL client
+
+A library for running server-side graphQL queries and mutations in Keystone. It replaces the `keystone.createItems` method with a set of utility functions to generate and execute graphQL queries.
+
+It differs from running queries directly with `keystone.executeGraphQL` by abstracting away a lot of the GraphQL syntax. You provide these functions with a `listName` and the data execute the query and it returns items without the internal graphQL list names.
+
+These functions use a server-side graphQL client that is based on same schema as your public API. You can also pass a `schemaName` to execute queries against a different schema including the private `internal` schema if you want to bypass access control checks.
+
+You can pass `extraContext`, which is added to the apollo context object, allowing you to execute queries as an authenticated user or provide any additional context for custom resolvers.
+
+These utilities can be used for a wide range of specific use-cases, some more common examples might include simple data seeding:
+
+```js
+const seedUsers = async usersData => {
+  await createItems({ keystone, listName: 'User', items: usersData });
+};
+```
+
+or fetching data inside hooks:
+
+```js
+// This example will copy data from a related field if set
+keystone.createList('Page', {
+  fields: {
+    name: { type: Text },
+    content: { type: Text },
+    copy: { type: Relationship, ref: 'Page' },
+  },
+  hooks: {
+    resolveInput: async ({ resolvedData }) => {
+      // Whenever copy field is set fetch the related data
+      const pageToCopy = resolvedData.copy
+        ? await getItem({
+            keystone,
+            listName: 'Page',
+            itemId: resolvedData.copy,
+            returnFields: 'name, content',
+          })
+        : {};
+      // resolve data from the copied item and unset the relationship
+      return { ...resolvedData, ...pageToCopy, copy: undefined };
+    },
+  },
+});
+```
+
+## API
+
+To perform CRUD operations, use the following functions:
+
+- [`createItem`](#createitem)
+- [`createItems`](#createitems)
+- [`updateItem`](#updateitem)
+- [`updateItems`](#updateitems)
+- [`deleteItem`](#deleteitem)
+- [`deleteItems`](#deleteitems)
+
+For custom queries use [`runCustomQuery`](#runcustomquery).
+
+- `createItem`
+- `createItems`
+- `updateItem`
+- `updateItems`
+- `deleteItem`
+- `deleteItems`
+
+For custom queries use `runCustomQuery`
```

Comment:
```suggestion
```

Reply:
Yep bad merge, looks like Aman and I updated the same thing